ml/Config.cc

```diff
@@ -22,7 +22,7 @@ static T clamp(const T& Value, const T& Min, const T& Max) {
 void Config::readMLConfig(void) {
     const char *ConfigSectionML = CONFIG_SECTION_ML;
 
-    bool EnableAnomalyDetection = config_get_boolean(ConfigSectionML, "enabled", false);
+    bool EnableAnomalyDetection = config_get_boolean(ConfigSectionML, "enabled", true);
```

Comment:
Most of the changes in this file are meant to make testing easy and they will be reverted prior to merging the PR. ML is enabled and detection/prediction runs every minute just for the `system.cpu` chart. The anomaly rates are updated every 5 seconds.